consoles/mesos.html

```diff
@@ -0,0 +1,61 @@
+{{ template "head" . }}
+
+{{ template "prom_right_table_head" }}
+<tr>
+    <th colspan="2">Statistics</th>
+</tr>
+<tr>
+  <td>Mesos Jobs (active)</td>
```

Comment:
In line with the other consoles, this should start with up/total mesos exporters